showkase-processor-testing/build.gradle

```diff
@@ -76,24 +74,8 @@ dependencies {
     androidTestImplementation deps.test.androidXTestCore
     androidTestImplementation deps.test.androidXTestRules
     androidTestImplementation deps.test.androidxTestRunner
-    // Allows this module to access the annotation processor related classes. Otherwise those are 
-    // only available in java library modules. Inspiration - 
-    // https://github.com/airbnb/epoxy/blob/master/epoxy-processortest/build.gradle
-    testImplementation files(getRuntimeJar())
-    testImplementation files(Jvm.current().getToolsJar())
+    // Allows this module to access the annotation processor related classes. Otherwise those are
+    // only available in java library modules.
+    testImplementation files('libs/rt.jar')
```

Comment:
:( I was hoping we don't have to use this option of adding a Java 8 jar file separately in a Java 11 project

Reply:
yea there is no other option as of now. we can create a ticket for Android group and ask them to add jmods folder in jdk provided by android studio

Reply:
Orcale jdk does provide it.

Reply:
ah I see, interesting! Better than the Java8 option I guess.